### PR TITLE
Feature/style

### PR DESCRIPTION
**Pull request by Mulloa90**

### CODE REVIEW

File: controllers/homeRoutes.js [hunk 1]
```diff
@@ -23,12 +27,6 @@ router.get('/goals/:goal_id', async (req, res) => {
   }
 });
 
-
-
-
-
-module.exports = router;
```
Comment (aurumbright) [inline controllers/homeRoutes.js]: Did you delete module.exports = router entirely from this route? Should definitely stay in there.

Comment (Mulloa90) [reply controllers/homeRoutes.js]: My apologies

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -1,55 +1,65 @@
-<!DOCTYPE html>
```
Comment (aurumbright) [inline views/layouts/main.handlebars:1]: @kentshaffer Do you want to review this one to make sure your toggle bar is still compatible?

File: controllers/homeRoutes.js [hunk 3]
```diff
@@ -2,15 +2,19 @@ const router = require('express').Router();
 const { Goal, Todo } = require('../models');
 // const withAuth = require('../utils/auth');
 
+router.get('/', async (req, res) => {
+  res.render('homepage', { logged_in: req.session.logged_in });
```
Comment (aurumbright) [inline controllers/homeRoutes.js:7]: @Mulloa90 Do we need to set 'login' if not logged in or does it render as the default?

**Review by aurumbright — APPROVED** [review]

**Pull request merged**